CHANGELOG.md

```diff
@@ -2,10 +2,17 @@
 
 All notable changes to the "azure-cosmosdb" extension will be documented in this file.
 
+## 0.15.0 - 2020-09-14
+### Added
+- PostgreSQL support to Attached Accounts
```

Comment:
I feel like these aren't quite readable enough. Looking at previous entries and modeling after that can be helpful. For example, a previous one says:

> Attach Cosmos DB accounts by connection string

so we might do:

> Attach PostgreSQL servers by connection string

Another example:

> API support for integration with [App Service extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azureappservice)

So we could do:

> PostgreSQL support for integration with [App Service extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azureappservice)